### PR TITLE
Adding support to scan clair-scanner json data.

### PR DESCRIPTION
**Pull request by amandel**

### CODE REVIEW

File: src/main/java/edu/hm/hafner/analysis/parser/ClairParser.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.ReaderFactory;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+/**
+ * A parser for dockerlint json output.
```
Comment (uhafner) [inline src/main/java/edu/hm/hafner/analysis/parser/ClairParser.java:21]: Wrong documentation (copy paste )

File: src/main/java/edu/hm/hafner/analysis/parser/ClairParser.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    public boolean accepts(final ReaderFactory readerFactory) {
+        return readerFactory.getFileName().endsWith(".json");
+    }
+
+    @Override
+    public Report parse(final ReaderFactory readerFactory) throws ParsingException {
+        Report report = new Report();
+        try (Reader reader = readerFactory.create()) {
+            JSONObject jsonReport = (JSONObject) new JSONTokener(reader).nextValue();
+            image = optStringIgnoreCase(jsonReport,"image");
```
Comment (uhafner) [inline src/main/java/edu/hm/hafner/analysis/parser/ClairParser.java:42]: Better as local variable that is an additional parameter to `convertToIssue`

File: src/main/java/edu/hm/hafner/analysis/parser/ClairParser.java (hunk 1 above)
Comment (amandel) [reply src/main/java/edu/hm/hafner/analysis/parser/ClairParser.java:21]: Sorry :(

```suggestion
 * A parser for clair scanner json output.
```

File: src/main/java/edu/hm/hafner/analysis/parser/ClairParser.java (hunk 2 above)
Comment (amandel) [reply src/main/java/edu/hm/hafner/analysis/parser/ClairParser.java]: I've just committed the change.

**Comment by uhafner** [comment]
Can you please also have a look at the build results, there are a couple of new warnings there:
https://ci.jenkins.io/job/Plugins/job/analysis-model/job/PR-468/7/

**Comment by amandel** [comment]
This should be it.

**Comment by uhafner** [comment]
Thank you very much!

**Pull request merged**